modules/protocol/unreal4.cpp

```diff
@@ -1493,6 +1740,13 @@ class ProtoUnreal : public Module
 
 		return EVENT_CONTINUE;
 	}
+
+	void OnChannelUnban(User *u, Channel *c) anope_override
+	{
+		UplinkSocket::Message(Config->GetClient("ChanServ")) << "SVS2MODE " << c->name << " -b " << u->GetUID();
```

Comment:
If the UNBAN command is assigned to a different service (weird I know, but its possible) this might result in an empty source.

You could change this event to take a `ChannelInfo*` instead of a `Channel*` and then use `ci->WhoSends()`.